functions/funtions.py

```diff
@@ -104,24 +103,24 @@ def predict(
     """Predict the ratings to movies
 
     Args:
-        test_set_origin (numpyarray)
-        x_id (int)
-        x_rated (numpyarray)
-        S (numpyarray)
-        k (int)
-        min_k (int)
-        x_list (numpyarray)
-        y_list (numpyarray)
-        global_mean (float)
-        true_testset_movies_id (numpyarray)
-        bu (numpyarray)
-        bi (numpyarray)
-        pu (numpyarray)
-        qi (numpyarray)
-        algorithm (str, optional). Defaults to "MF".
+        test_set_origin (numpyarray): storing all user/item pairs we want to predict the ratings.
+        x_id (int): the user_id we want to recommend to him.
+        x_rated (numpyarray): All users who rated each item are stored in list.
+        S (numpyarray): Compute the similarity between all pairs of users.
+        k (int): Number of neighbors use in prediction
```

Comment:
```suggestion
        k (int): Number of neighbors use in prediction.
```
Either every sentence end with a dot or not.
It should be consistent across all the docstring.